tests/resources/configs/loader.error/test.conf

```diff
@@ -1 +1 @@
-include file(url("http://exampe.com"))
+include file(url("http://example.com"))
```

Comment:
Note: there are specific RFC domains available for examples. If you're intentionally trying to make an example of something, one should use one of them instead of a domain which is currently for sale and could randomly gain content.